docs/integrations/mlstudio_designer.md

```diff
@@ -0,0 +1,147 @@
+# Azure Machine Learning Designer Integration
+
+Connecting to the Feature Store from the Azure Machine Learning Designer requires setting up a Feature Store API key for the Designer and installing the **HSFS** on the Designer. This guide explains step by step how to connect to the Feature Store from Azure Machine Learning Designer.
```

Comment:
**HSFS** python library on the Designer

Reply:
I don't really get that comment. ** is a formatting